### ISSUE TITLE
Potential error in documentation string of isolated_labels method

### ISSUE THREAD

**Issue by Hrovatin** [theislab/scib#148]
In the documentation string of isolated_labels method it says that "if n=None, consider any label that is missing at least 1 batch".https://github.com/theislab/scib/blob/b2784f5bb0f5c39c42b928957e106c53c5ba4067/scIB/metrics.py#L314
If I understand correctly, this should consider a cell type that is present in even 12 out of 13 batches as isolated, right? However, if I run the method get_isolated_labels (that uses the n from isolated_labels) with n=None I get the output "isolated labels: no more than 1 batches per label". There seems to be a potential error in how the n=None case is dealt with in get_isolated_labels https://github.com/theislab/scib/blob/b2784f5bb0f5c39c42b928957e106c53c5ba4067/scIB/metrics.py#L344 based on the documentation of the isolated_labels method.

**Comment by LuckyMD** [comment]
Hey @Hrovatin, 

Thanks for pointing out the documentation issue here. The idea is that `n=None` should return the label shared by the fewest batches. Do you know what you were trying to say with this wording, @mumichae? We should adapt this ;).

**Comment by mumichae** [comment]
True, I didn't update the documentation for this function properly. It is fixed now in 68e86a7e0be465c1b7effb5ed190135f9a01c6ff awaiting review. I hope it is clearer now.

```
n: max number of batches per label for label to be considered as isolated.
            if n is integer, consider labels that are present for n batches as isolated
            if n=None, consider minimum number of batches that labels are present in
```

If not, please let me know.
